### PR TITLE
Remove `use_buffers` from `PatchDoc` message

### PR DESCRIPTION
**Pull request by mattpap**
@bryevdv, I'm not sure what was the point of `use_buffers=False`, but it seems it isn't needed anymore.

### CODE REVIEW

File: bokeh/client/connection.py [hunk 1]
```diff
@@ -388,14 +388,11 @@ def have_reply_or_disconnected() -> bool:
...
     def _send_patch_document(self, session_id: ID, event: DocumentChangedEvent) -> None:
         # XXX This will cause the client to always send all columns when a CDS
-        # is mutated in place. Additionally we set use_buffers=False below as
-        # well, to suppress using the binary array transport. Real Bokeh server
-        # apps running inside a server can handle these updates much more
-        # efficiently
+        # is mutated in place.
         from bokeh.document.events import ColumnDataChangedEvent
         if isinstance(event, ColumnDataChangedEvent):
             event.cols = None
```
Comment (mattpap) [inline bokeh/client/connection.py:394]: @bryevdv, what's the point of this? The comment doesn't explain anything.

Comment (bryevdv) [reply bokeh/client/connection.py:394]: Buffers originally only worked from server to browser, and not from client session to browser. The real bokeh apps was in reference to the practice (at the time) of some users trying to make apps “outside” the server, with session.loop_until_closed

Comment (mattpap) [reply bokeh/client/connection.py:394]: So, I can remove this case as well?

**Review by bryevdv — APPROVED** [review]

**Comment by bryevdv** [comment]
> I'm not sure what was the point of use_buffers=False, but it seems it isn't needed anymore.

Originally buffers were only supported server -> browser and nowhere else, and this flag was used to suppress attempts to use them in places they would not function.

**Pull request merged**